specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2015-11-01-preview/LinkedServices.json

```diff
@@ -0,0 +1,325 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Log Analytics",
+    "description": "Azure Log Analytics API reference",
+    "version": "2019-08-01-preview"
```

Comment:
Hi. Is there any reason about this one?

Reply:
it's a 2019-08-01-preview version. But the folder is 2015-11-01-preview

Reply:
Fixed